website/public/install/common-manual-tabs.ejs.html

```diff
@@ -0,0 +1,148 @@
+<%#
+<!-- Copyright (C) 2020  Matthew "strager" Glazar -->
+<!-- See end of file for extended copyright information. -->
+%>
+
+<style>
+  .show-with-js {
+    display: none;
+  }
+
+  .js-enabled .show-with-js {
+    display: block;
+  }
+
+  .hide-with-js {
+    display: block;
+  }
+
+  .js-enabled .hide-with-js {
+    display: none;
+  }
+
+  .tab-bar a {
+    display: inline-block;
+    padding: 6px 12px;
+    background-color: #aaaaaa;
+    transition: 0.3s;
+    font-size: medium;
+    color: rgb(0, 0, 0);
+
+    border-bottom-color: rgb(156, 23, 23);
+
+    -webkit-user-select: text;
+    -moz-user-select: text;
+    -ms-user-select: text;
+    user-select: text;
+  }
+
+  .tab-bar {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+  }
+
+  .tab-bar li {
+    float: left;
+  }
+
+  .tab-bar li a {
+    display: block;
+    text-align: center;
+    padding: 16px;
+  }
+
+  .js-enabled .tab-bar a {
+    text-decoration: none;
+  }
+
+  .tab-bar a:hover {
+    background-color: #858585;
+  }
+
+  .js-enabled .tab-bar li {
+    border-bottom: 4px solid;
+    border-bottom-color: black;
+  }
+
+  .js-enabled .tab-bar li.tab-bar-selected {
+    border-bottom-color: rgb(255, 0, 0);
+  }
+
+  @media (prefers-reduced-motion) {
+    .tab-bar a {
+      transition: 0ms;
+    }
+  }
+
+  .package-manager {
+    border-left: 1px #858585 solid;
+    padding-left: 8px;
+  }
+
+  .hidden.hidden {
+    display: none;
+  }
+
+  @media (prefers-color-scheme: dark) {
+    .tab-bar a {
+      color: rgb(0, 0, 0);
+      background-color: #bdbdbd;
+    }
+  }
+</style>
+
+<script>
+  document.body.classList.add("js-enabled");
+  changePackageManager(null, "manual-nothing-selected");
+
+  function changePackageManager(event, selectedManager) {
+    let tabContent = document.getElementsByClassName("package-manager");
+    for (let i = 0; i < tabContent.length; i++) {
+      tabContent[i].classList.add("hidden");
+    }
+
+    let tabBarButtons = document.getElementsByClassName("tab-bar-button");
+    for (let i = 0; i < tabBarButtons.length; i++) {
+      tabBarButtons[i].classList.remove("tab-bar-selected");
+    }
+
+    if (selectedManager === "none") {
```

Comment:
Shouldn't this be `"manual-nothing-selected"` to match line 98? Or is this for something else?

Reply:
Assuming you mean the `selectedManager === ` section in this snippet, it's a part of some old code I don't use anymore. I have removed it.